<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Transformer Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" rowspan="2">
     <widget class="QSmartGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>560</width>
        <height>500</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(89, 89, 89);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>330</height>
       </size>
      </property>
      <property name="title">
       <string>Transformation</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_original">
         <property name="text">
          <string>Original</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_wave">
         <property name="text">
          <string>Wave</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_tilt">
         <property name="text">
          <string>Tilt</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Circle</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Hough Transform</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_houghLine">
           <property name="text">
            <string>Line</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_houghRectangle">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Rectangle</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Line Threshold</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_lineThresholdValue">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>70</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_lineThreshold">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>70</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionOpen_image"/>
    <addaction name="actionImport_from_Main"/>
    <addaction name="actionExport_to_Main"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFiles"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_image">
   <property name="text">
    <string>Open image</string>
   </property>
  </action>
  <action name="actionImport_from_Main">
   <property name="text">
    <string>Import from Main</string>
   </property>
  </action>
  <action name="actionExport_to_Main">
   <property name="text">
    <string>Export to Main</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSmartGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qsmartgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
